roles/lgsm/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+########################################################################
+# Title:            Sandbox: LGSM | Multi-instance Tasks               #
+# Author(s):        scarabyte                                          #
+# URL:              https://github.com/saltyorg/Sandbox                #
+# --                                                                   #
+########################################################################
+#                   GNU General Public License v3.0                    #
+########################################################################
+---
+- name: Ensure we have instances for lgsm to install
+  ansible.builtin.assert:
+    that:
+      - lgsm_instances is defined
+      - lgsm_instances is not string
+      - lgsm_instances is not mapping
+      - lgsm_instances is iterable
+      - lgsm_instances | length > 1
+    fail_msg: >-
+      LGSM was designed to use multi-instance configuration to install game servers properly. Check out https://docs.saltbox.dev/reference/multiple-instances/
+    success_msg: >-
+      Detected multi-instance configuration.
```

Comment:
Let's just check if the list has items and direct to the docs if not

Reply:
Example could be something like
```yaml
- name: Ensure we have instances for lgsm to install
  ansible.builtin.assert:
    that:
      - (lgsm_instances | type_debug) == "list"
      - lgsm_instances | length > 0
    fail_msg: >-
      Check the docs for how to configure this role: <url>
    success_msg: >-
      Detected multi-instance configuration.
```

Reply:
that being said the 
```yaml
      - lgsm_instances is not string
      - lgsm_instances is not mapping
      - lgsm_instances is iterable
```
bits are fine, defined and length are the ones that need changes.